### PR TITLE
Introduce admin consent

### PR DESCRIPTION
**Pull request by MrGeneration**

### CODE REVIEW

File: channels/microsoft365/accounts/register-app.rst [hunk 1]
```diff
@@ -119,5 +119,57 @@ To get started, head over to `Microsoft’s Azure Portal`_.
...
+      select *App registrations*. Within your desired app go to
+      *API permissions* and use the *Grand admin consent for {company name}*
+      button to generally allow connecting users you previously consented.
+
+      .. note::
+
+         Not adding users / groups and providing the granted admin consent
+         right away will cause Microsoft to enforce at least the first user
+         to be requested to provide a reason for the consent request. After that
+         Microsoft will automatically grand the consent for your tenant.
```
Comment (fliebe92) [inline channels/microsoft365/accounts/register-app.rst:168]: `grand` => `grant`

File: channels/microsoft365/accounts/register-app.rst [hunk 2]
```diff
@@ -119,5 +119,57 @@ To get started, head over to `Microsoft’s Azure Portal`_.
...
+         manually.
+
+      Still within *Enterprise applications* select *Users and groups*.
+      In this section you can either specific users or groups
+      (users must be direct members!) that are allowed to use your app for
+      adding mailboxes to Zammad.
+
+      After adding users and groups, go back to the Azure portals home and
+      select *App registrations*. Within your desired app go to
+      *API permissions* and use the *Grand admin consent for {company name}*
```
Comment (fliebe92) [inline channels/microsoft365/accounts/register-app.rst:160]: `Grand` => `Grant`

File: channels/microsoft365/accounts/account-setup.rst [hunk 3]
```diff
@@ -36,12 +36,75 @@ in third-party email clients (like Zammad).
...
+      :scale: 50%
+      :align: center
+
+Granting the consent (admin users)
+   Within **Enterprise applications** go to **Admin consent requests**.
+   You'll find all your user grand requests here.
+
+   Clicking on a request allows you to review which user requested the consent
+   for what app. Click on **Rewview permissions and consent** to start the
+   grand process. You'll be asked for credentials - these credentials are the
```
Comment (fliebe92) [inline channels/microsoft365/accounts/account-setup.rst:73]: `grand` => `grant`

File: channels/microsoft365/accounts/account-setup.rst [hunk 4]
```diff
@@ -36,12 +36,75 @@ in third-party email clients (like Zammad).
...
+   administrator credentials *not* the ones of the mailbox user to add.
+
+   After you've granted the request, continue with `Add a New Account`_.
+
+   .. figure:: /images/channels/microsoft365/accounts/account-setup/granting-consent-request-of-user.gif
+      :alt: Within Enterprise applications grand requests within admin consent
```
Comment (fliebe92) [inline channels/microsoft365/accounts/account-setup.rst:79]: `grand` => `grant`

**Review by fliebe92 — COMMENTED** [review]
@MrGeneration I really love your changes! 😍 I just marked some typos and one sentence that I don't understand (but maybe it's me... 🤷 )

Thanks a lot for your effort!

File: channels/microsoft365/accounts/register-app.rst [hunk 5]
```diff
@@ -119,5 +119,57 @@ To get started, head over to `Microsoft’s Azure Portal`_.
...
+         changes (e.g. requesting more permissions than originally noted).
+
+         It also has another benefit: You can add the email account in question
+         immediately without any administrator being forced to stepped in
+         manually.
+
+      Still within *Enterprise applications* select *Users and groups*.
+      In this section you can either specific users or groups
+      (users must be direct members!) that are allowed to use your app for
+      adding mailboxes to Zammad.
```
[the remark this answers is not in the document]
Comment (MrGeneration) [reply channels/microsoft365/accounts/register-app.rst:156]: Now that I'm more awake than yesterday night I even felt that the wording can be better - fix is on the way.

**Comment by MrGeneration** [comment]
Thank you so much, you're indeed right. Could you please check again? 🙏

**Comment by fliebe92** [comment]
> Thank you so much, you're indeed right. Could you please check again? 🙏

Looks great! 🚀

**Review by fliebe92 — APPROVED** [review]

**Pull request merged**